test/realtime/channelgroup.test.js

```diff
@@ -0,0 +1,701 @@
+'use strict';
```

Comment:
I realized that test files are not automatically picked up by the playwright browser tests as they are for node.
If you check, for example, [this CI run for webkit](https://github.com/ably/ably-js/actions/runs/8177130978/job/22358085711) in this PR you can see there are no `realtime/channelgroup` tests completed.

It seems that we need to add new test file names to `test/support/browser_file_list.js` for it to be included in browser tests. I hope it's the only place we need to change for that.

You can check locally if it works if you do `describe.only('realtime/channelgroup'` below, and run `PLAYWRIGHT_BROWSER=chromium npm run test:playwright` (might need to do `npx playwright install --with-deps` before that to install browsers)